telegram/_story.py

```diff
@@ -18,24 +18,65 @@
 # along with this program.  If not, see [http://www.gnu.org/licenses/].
 """This module contains an object related to a Telegram Story."""
 
-from typing import Optional
+from typing import TYPE_CHECKING, Optional
 
+from telegram._chat import Chat
 from telegram._telegramobject import TelegramObject
 from telegram._utils.types import JSONDict
 
+if TYPE_CHECKING:
+    from telegram import Bot
+
 
 class Story(TelegramObject):
     """
-    This object represents a message about a forwarded story in the chat. Currently holds no
-    information.
+    This object represents a story.
+
+    Objects of this class are comparable in terms of equality. Two objects of this class are
+    considered equal, if their :attr:`chat` and :attr:`id` are equal.
 
     .. versionadded:: 20.5
 
+    .. versionchanged:: NEXT.VERSION
+        Added attributes ``chat`` and ``id`` and equality based on them.
+
+    Args:
+        chat (:class:`telegram.Chat`): Chat that posted the story.
+        id (:obj:`int`): Unique identifier for the story in the chat.
+
+    Attributes:
+        chat (:class:`telegram.Chat`): Chat that posted the story.
+        id (:obj:`int`): Unique identifier for the story in the chat.
```

Comment:
missing versionadded's

Reply:
I thought above is enough, we did the same somewhere else I copied this from

Reply:
tbh I don't like the versionchanged mentioning that